### DISCUSSION TITLE
Go2Rtc LXC version update

### DISCUSSION THREAD

**Discussion by brusarp** [tteck/Proxmox#1440] [Help Needed] [answered]
Greetings,
I need help updating the go2rtc LXC build with related script.
Now my version is 1.2 but on go2rtc's github it's updated to 1.5.
I use the same script on the LXC console for the update. No errors are displayed but the version of go2rtc is still 1.2.

How to upgrade to 1.5?

Thank you!
Andrew

**Comment by tteck** [comment, marked as answer]
Try this:
```
cd /opt/go2rtc
rm go2rtc_linux_amd64
wget -q https://github.com/AlexxIT/go2rtc/releases/latest/download/go2rtc_linux_amd64
chmod +x go2rtc_linux_amd64
```

**Reply by brusarp** [reply]
Good, thank you!
I also had to restart the LXC (obviously)...

**Reply by tteck** [reply]
Executing the script within the LXC console now functions correctly for the update process (without reboot).

**Reply by brusarp** [reply]
Thank you very much!
